wiki/osu!_File_Formats/Osu_(file_format)/en.md

```diff
@@ -194,26 +194,58 @@ Hit Circle Syntax:
 `x,y,time,type,hitSound,addition`
 `164,260,2434,1,0,0:0:0:0:`
 
+x (Integer) ranges from 0 to 512 (inclusive) and y (Integer) ranges from 0 to 384 (inclusive).
+
+time (Integer) is in milliseconds from the beginning of the song.
+
+type (Integer) is a bitmap:
+
+1 = Circle, 2 = Slider, 4 = New combo, 8 = Spinner. More significant bits can be used to choose the next combo color.
```

Comment:
It's probably worth explaining how the combo color skips work. It can be as simple as "Bits 5 to 7 tell how many combo colors will be skipped, if the object is also a new combo."

Also 128 = osu!mania Hold